### PR TITLE
feat: add progress bar component

### PR DESCRIPTION
**Pull request by jenseo**

### CODE REVIEW

File: src/components/cards/ProgressBar/ProgressBar.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const Template: Story<ProgressBarProps> = (args) => <ProgressBar {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  children: (
+    <>
+      <Paragraph>Primary</Paragraph>
+      <Paragraph color="secondary">Secondary</Paragraph>
+    </>
+  ),
```
Comment (filiptammergard) [inline src/components/cards/ProgressBar/ProgressBar.stories.tsx:19]: Should be removed, right?

**Review by filiptammergard — APPROVED** [review]
Sweet!

**Comment by filiptammergard** [comment]
Do you have access to looking into the details on the UI Review and UI Tests checks?

**Comment by jenseo** [comment]
> Do you have access to looking into the details on the UI Review and UI Tests checks?

Yes, it seems so now!

**Comment by jenseo** [comment]
@filiptammergard it seems I only have read rights though, so I'm not sure I can approve it if I need to click some button somewhere :)

**Comment by filiptammergard** [comment]
> @filiptammergard it seems I only have read rights though, so I'm not sure I can approve it if I need to click some button somewhere :)

I sent an invite to Chromatic now 🙂

**Pull request merged**